packages/next/src/client/components/react-dev-overlay/server/middleware.ts

```diff
@@ -318,6 +302,12 @@ function getOverlayMiddleware(options: OverlayMiddlewareOptions) {
           Boolean(parseInt(frame.lineNumber?.toString() ?? '', 10))
         )
       ) {
+        if (sourcePackage) {
```

Comment:
would be nice to group them into one place later, not blocking

Reply:
Yeah, I think the whole `middleware.ts` could use a bit of refactoring.